atlasclient/models.py

```diff
@@ -846,3 +846,62 @@ class SearchFulltext(base.QueryableModel):
     relationships = {'entities': Entity,
                      'attributes': AttributeDef,
                      'fullTextResults': FullTextResult}
+
+
+class SearchSavedCollection(base.QueryableModelCollection):
+    """We can also remove this part but just keeping same structure as above"""
+    def load(self, response):
+        for item in response:
+            model = self.model_class(self,
+                                     href=item.get('href'))
+            model.load(item)
+            self._models.append(model)
+
+
+class SearchSaved(base.QueryableModel):
+    collection_class = SearchSavedCollection
+    path = 'search/saved'
+    data_key = 'search_saved'
+    primary_key = 'guid'
+    fields = ('name', 'searchParameters', 'ownerName', 'searchType', 'uiParameters')
+
+    def inflate(self):
```

Comment:
any particular reason you are overriding the inflate function here ?

Reply:
Here in the base mode an additional check for searchParameter is not required for searchsaved and it was actually preventing the response to inflate. So I removed below part in the override.
`params = self.searchParameters if hasattr(self, 'searchParameters') else {}`

Reply:
but we do need `self.searchParameters` if available, right ?